src/main/java/net/sf/jabref/logic/util/io/SimpleFileListEntry.java

```diff
@@ -0,0 +1,94 @@
+/*  Copyright (C) 2015 JabRef contributors.
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+package net.sf.jabref.logic.util.io;
+
+import java.util.List;
+
+/**
+ * This class represents a simplified version of the file link for a Bibtex entry.
+ */
+public class SimpleFileListEntry {
```

Comment:
A `FileListEntry`, as used in `FileListTableModel` has an `ExternalFileType` as the third parameter. This results in bindings to the GUI package etc. A `SimpleFileListEntry` has the "correct" type name (as specified by an existing `ExternalFileType`) as the third parameter, which is enough for serialization but not for GUI.

As stated above, the difference between a plain `List<SimpleFileListEntry>` and `SimpleFileList` is that `ExternalFileType` is consulted via the preferences when the entries are generated and that it has methods required for simple drop-in replacement of `FileListTableModel`, including serialization.

Yes, maybe these methods should go to `model`. However, it is not clear exactly what should go where. Also, looking at the time perspective, it wasn't obvious from the start that I would end up here. Finally, there's a bit of discussion how many methods should be added to the entries. Now, parts of this is rather JabRef specific and should maybe stay away from `model` (which I think is aimed to be as "clean" as possible). 

Naming: yes, I agree that they are not exceptionally well chosen. One should also take into consideration that `FileListEntry` now extends`SimpleFileListEntry`, and that there's a `FileListEntryEditor` (and a `FileListTableModel`).